lib/queue_classic_matchers.rb

```diff
@@ -132,6 +132,8 @@ def amount
 
     results.any? do |entry|
       time_matches = if @time
+        next entry['scheduled_at'] if entry['scheduled_at'].is_a?(Time) # ActiveRecord 6
```

Comment:
The line below returns a boolean while `next entry['sheduled_at']` will return a data - are we sure that's what we want?

Reply:
This is plain wrong 🤕 Fixing!

Reply:
@kruszczynski thanks for spotting that! Fixed on the `rails6` branch, but the PR won't update because of github slowness